contributors/devel/api-conventions.md

```diff
@@ -555,172 +550,7 @@ below.
-default:
-
-```yaml
-containers:
-  - name: nginx
-    image: nginx-1.0
-  - $patch: replace   # any further $patch operations nested in this list will be ignored
-```
-
-To delete an element of a list that should be merged:
-
-```yaml
-containers:
-  - name: nginx
-    image: nginx-1.0
-  - $patch: delete
-    name: log-tailer  # merge key and value goes here
-```
-
-Delete operation will delete the first entry in the list that match the merge key.
-But there is validation to make sure no 2 entries with the same merge key will get in the list.
-
-#### List of Primitives
-
-We have two patch strategies for lists of primitives: replace and merge.
-Replace is the default patch strategy, which will replace the whole list on update and it will preserve the order;
-while merge strategy works as an unordered set. In this section, we call a primitive list with merge strategy an unordered set.
-The patch strategy is defined in the go struct tag of the API objects,
-e.g. `finalizers` uses `merge` as patch strategy.
-```go
-Finalizers []string `json:"finalizers,omitempty" patchStrategy:"merge" protobuf:"bytes,14,rep,name=finalizers"`
-```
-
-##### Unordered Set
-
-There are 3 operations: add, delete, replace.
-
-Suppose we have defined a `finalizers` and we call it the original finalizers:
-
-```yaml
-finalizers:
-  - a
-  - b
-  - c
-```
-
-1) To add items in a set, we use the list name as the key.
-e.g. to add items "d" and "e" in the original finalizers, the patch will be:
-
-```yaml
-finalizers:
-  - d
-  - e
-```
-
-After applying the patch on the original finalizers, it will become:
-
-```yaml
-finalizers:
-  - a
-  - b
-  - c
-  - d
-  - e
-```
-
-2) To delete items in a set, we use a parallel list with key: `$deleteFromPrimitiveList/\<keyOfPrimitiveList\>`.
-e.g. to delete items "b" and "c" from the original finalizers, the patch will be:
-
-```yaml
-$deleteFromPrimitiveList/finalizers:
-  - b
-  - c
-```
-
-After applying the patch on the original finalizers, it will become:
-
-```yaml
-finalizers:
-  - a
-```
-
-In an erroneous case, the set may be created with duplicates. Deleting an item that has duplicates will delete all matching items.
-
-3) Replace can be fulfilled by an addition and a deletion.
-e.g. to replace "a" with "x" in the original finalizers, the patch will be:
-
-```yaml
-$deleteFromPrimitiveList/finalizers:
-  - a
-finalizers:
-  - x
-```
-
-After applying the patch on the original finalizers, it will become:
-
-```yaml
-finalizers:
-  - x
-  - b
-  - c
-```
-
-### Map Operations
-
-To indicate that a map should not be merged and instead should be taken literally:
-
-```yaml
-$patch: replace  # recursive and applies to all fields of the map it's in
-containers:
-- name: nginx
-  image: nginx-1.0
-```
-
-To delete a field of a map:
-
-```yaml
-name: nginx
-image: nginx-1.0
-labels:
-  live: null  # set the value of the map key to null
-```
-
+Details of Strategic Merge Patch is in this [doc](../strategic-merge-patch.md).
```

Comment:
> Details of Strategic Merge Patch is in this [doc](../strategic-merge-patch.md).

How about:

Details of Strategic Merge Patch are covered [here](../strategic-merge-patch.md).